### ISSUE TITLE
Unwanted page break

### ISSUE THREAD

**Issue by guilhermeagirardi** [motla/vue-document-editor#27]
Hi @motla 

I don't want this page break to happen for my sections.

I am unable to prevent this behavior via CSS alone, as it seems that vue-document-editor does not seem to take into consideration page-break-before, page-break-after or page-break-inside. What do I need to do?

Ps.: My sections are in `<section>` elements.

![image](https://user-images.githubusercontent.com/2306117/230999617-71a7af77-3d0b-4116-9b46-270dc1bb39f8.png)

**Comment by motla** [comment]
Hi @guilhermeagirardi,
It is not implemented yet and not trivial to implement. For example we have to handle the case when the section height is higher than the page height. I can't look at it right now, but the starting point would be to detect the `<section>` element in this line to avoid splitting it:

https://github.com/motla/vue-document-editor/blob/8eaa28b98e1d01274c35415e8e283c4b1975baa3/src/DocumentEditor/imports/page-transition-mgmt.js#L53

**Comment by guilhermeagirardi** [comment]
Hi @motla !

Thank you for the feedback and the suggestion.

What I did was add a callback at the end of the conditional test of that function.

```js
else if (!sub_child.childNodes.length || sub_child.tagName.match(/h\d/i) || do_not_break(sub_child)) {
```

```vue
<vue-document-editor
    ref="editor"
    v-model:content="content"
    :do_not_break="(sub_child) => sub_child.tagName.match(/section/i)"
/>
```

This was enough for me at the time to define different breakage prevention situations.

**Comment by motla** [comment]
Well done! Great implementation 👍
I may add it to the next release but I'm afraid of edge cases, it needs some testing.

**Comment by motla** [comment]
Hi @guilhermeagirardi 
For your information I just released v2.3.0 (Vue3) and v1.5.0 (Vue2) which contains your implementation. I have set you as co-author.

**Comment by guilhermeagirardi** [comment]
Hi @motla!

It was no big deal. I'm honored to know that. Thankss! 😁
